src/site/paragraphs/collapsible_panel.drupal.liquid

```diff
@@ -30,7 +30,7 @@
     <ul class="usa-unstyled-list">
         {% for accordionItem in entity.fieldVaParagraphs %}
             {% assign item = accordionItem.entity %}
-            <li data-entity-id="{{ item.entityId }}">
+            <li id="{{ item.entityId }}" data-entity-id="{{ item.entityId }}">
```

Comment:
Per convo, we updated the hash to only be the ID -

![image](https://user-images.githubusercontent.com/1915775/98047574-2386d300-1dfa-11eb-94b9-47698a42e806.png)